.github/workflows/linux.yml

```diff
@@ -126,3 +126,86 @@ jobs:
       - name: Run tests
         run: ctest --test-dir build --output-on-failure
 
+  linux_aarch64:
+    name: "aarch64 cross-compilation"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    container:
+      image: openquantumsafe/ci-ubuntu-jammy:latest
+    env:
+      OPENSSL_BRANCH: "openssl-3.1"
```

Comment:
If you build and install OpenSSL anyway, why not 3.2 / master? The tests would be more thorough then. An older OpenSSL might also be OK (testing less), but then you probably don't have to build it (& be faster and more eco-friendly).